addons/delivery/wizard/choose_delivery_carrier.py

```diff
@@ -67,10 +67,9 @@ def _compute_available_carrier(self):
     def _get_shipment_rate(self):
         vals = self.carrier_id.rate_shipment(self.order_id)
         if vals.get('success'):
-            if vals.get('warning_message'):
-                self.delivery_message = vals['warning_message']
+            self.delivery_message = vals.get('warning_message')
             self.delivery_price = vals['price']
-            self.display_price = vals['carrier_price']
```

Comment:
I don't think we should change the display price, as it would mean that it is useless

Reply:
We should porbably change it in the button_confirm function